tests/unit/test-cmd/test.sign.js

```diff
@@ -397,6 +397,17 @@ describe('sign', () => {
       }
     ));
 
+    it('throws unexpected errors', async () => {
+      const fakeAsyncFsReadFile = sinon.spy(async () => {
+        throw new Error('Unexpected fs.readFile error');
+      });
+      await assert.isRejected(
+        getIdFromSourceDir('fakeSourceDir', fakeAsyncFsReadFile),
```

Comment:
If you want to minimize the number of injected dependencies, then you could create a directory or file without the right permissions (since the the implementation doesn't handle the `EACCES` error). This makes the test a bit less artificial than injecting a fake `fs.readFile`.

Reply:
yeah, I was initially thinking of going for that strategy, but I would prefer to don't add this test to the ones skipped on windows (I would like to reduce the number of tests skipped on windows, not growing that list), and so I preferred to use this strategy instead.